### DISCUSSION TITLE
(DE) Transfer der Repositories von Danielperna84 auf SukramJ

### DISCUSSION THREAD

**Discussion by SukramJ** [SukramJ/aiohomematic#1929] [Announcements]
Moin,

nachdem Daniel(@danielperna84) schon seit längerer Zeit nicht mehr aktiv an der Entwicklung beteiligt war, haben wir nun den logischen Schritt vollzogen, die relevanten Repositories ([custom_homematic](https://github.com/SukramJ/custom_homematic/), [hahomematic](https://github.com/SukramJ/hahomematic/), [pydevccu](https://github.com/SukramJ/pydevccu/)) auf mich (@SukramJ ) zu übertragen.

Alle bestehenden URLs die bisher auf die Repositories von Daniel verwiesen behalten ihre Gültigkeit, da Github diese auf meine Repositories umleitet.
Solltet ihr Booksmarks im Browser haben, so macht es allerdings Sinn diese auch jetzt schon umzustellen (danielperna84 --> sukramj).

Das HACS Repository wird von HACS automatisch migriert (Crazy).

Bleibt mir also an dieser Stelle nur Daniel für die tolle Projektidee, die initiale Implementierung und als kompetenten Ansprechpartner zu bedanken. Ohne diese Unterstützung hätte ich mich nicht an eine CCU-Integration getraut.
Ich hoffe er bleibt uns bei der Community-Arbeit noch eine weile erhalten, auch wenn er, so viel muss ich verraten, keine Homematic Komponenten mehr einsetzt.

**Comment by danielperna84** [comment]
Vielen Dank für die lieben Worte. Mein Dank geht an dich zurück, denn du hast meinen "Prototypen" zu einer phantastischen Integration umgeschrieben. Auf solch ein hohes Niveau hätte ich sie niemals bringen können. Weiter so. 👍

**Comment by l-mb** [comment]
Vielen Dank an euch beide!

Homematic (IP) ist eine sehr relevante Komponente in meinem HA Setup und ich bin wirklich sehr dankbar für diese Integration.

**Comment by stbuerger** [comment]
1k Dank an @danielperna84 und @SukramJ für diese äußerst wichtige und technisch sehr sehr gute Integration.
Erst durch DIESE Integration ist HA für mich wirklich nützlich geworden.

**Comment by Isengo1989** [comment]
Vielen Dank an euch beide für die tolle Arbeit! :raised_hands:

**Comment by kwithus** [comment]
Auch von mir vielen Dank an Euch beide und alle anderen, die diese Integration ständig weiter verbessern.
Vielleicht schafft es die Integration ja irgendwann ins core, aber vielleicht bin ich bis dahin auch weg aus der HA Welt, wenn ELV sich weiter der Öffnung Richtung Matter verweigert.

**Comment by schwurbelli** [comment]
Vielen Dank euch für die tolle Arbeit, nachdem die CCU der Dreh- und Angelpunkt meines Hauses ist, macht diese Integration HA erst sinnvoll nutzbar.

**Comment by pdmetze** [comment]
Großes Dankeschön an euch! Mein Haus ist vollständig mit Homematic IP Wired Geräten ausgestattet, auch alte wired, funk und IP funk sind bei mir im Einsatz. Homematic ist also das Herz meiner Hausautomation. Ich steige aktuell von ioBroker auf Home Assistant um, was ohne eure Homematic Integration überhaupt nicht in Frage käme.

**Comment by banholzer** [comment]
Auch meinerseits herzlichen Dank für die Arbeit. Auch wenn mich die HomeMatic Geräte zuletzt ziemlich nerven, habe ich doch einige davon im Einsatz und bin daher auf diese Integration, die wirklich einwandfrei funtioniert, angewiesen.
